docs/DeveloperGuide.md

```diff
@@ -290,7 +291,29 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: Add contact**
+
+**MSS**
+1. User requests to add a person
+2. AddressBook adds the contact to the list
```

Comment:
Maybe change from AddressBook to ConTActs?

Reply:
Actually I am not sure if we are supposed to use the class name (`AddressBook`) for use cases. If there is no need to use class names, ConTActs makes sense. Maybe @DKHB0207 @sayglenn @yockcheng can help us out here.

Reply:
Since we specified that System is 'AddressBook' (Line 292) then lets keep it as AddressBook.

If we change the system to ConTActs, then every use case should also be changed accordingly

Reply:
Personally, I feel like ConTActs make more sense, because we are supposed to describe the interaction so I don't think we need to go into the specific class name.